configs/original-dst-cluster/README.md

```diff
@@ -0,0 +1,76 @@
+# Original destination cluster configuration and testing
+
+An original destination cluster forwards requests to the same destination
+the request was going to before being redirected to Envoy using an
+iptables REDIRECT rule.  `proxy_config.json` contains an example Envoy
+configuration demonstrating the use of an original destination
+cluster. `netns_setup.sh` and `netns_cleanup.sh` are provided as
+examples for setting up and cleaning up, respectively, a network
+namespace and the required iptables rule to redirect traffic to Envoy.
+
+# Setting up
+
+`netns_setup.sh` takes two arguments: the name of the new network
+namespace and the prefix that is to be redirected. Envoy listener port
+is set to 10000, which matches the configuration in
+`proxy_config.json`.
+
+This creates a network namespace `ns1` and redirects traffic from
+there to Envoy listening on port 10000 if the desination address of
+the traffic matches `173.194.222.0/24` :
+
+```
+sudo ./configs/original-dst-cluster/netns_setup.sh ns1 173.194.222.0/24
+``` 
+
+# Building and running Envoy
+
+Build Envoy with debug options, so that the behavior can be better
+observed from the logs:
+
+```
+bazel build --verbose_failures //source/exe:envoy-static -c dbg
```

Comment:
Nit: the `--verbose_failures` is a Bazel thing, you don't need it actually for logs or anything.